tests/lean/584a.lean.expected.out

```diff
@@ -1,5 +1,5 @@
-foo : Π (A : Type u_1) [H : inhabited A], A → A
-foo' : Π {A : Type u_1} [H : inhabited A] {x : A}, A
+foo : Π (A : Type u_1) [_ : inhabited A], A → A
+foo' : Π {A : Type u_1} [_ : inhabited A] {_ : A}, A
```

Comment:
I think it would make more sense to print anonymous instance binders as `[inhabited A]` instead of `[_ : inhabited A]`

Reply:
Good idea -- added that feature.